docs/sources/viewing-json-logs/_index.md

```diff
@@ -24,17 +24,28 @@ To interact with the JSON view, select the **Show Logs** button for your service
 
 {{< figure alt="JSON Table viewer with selector and include exclude highlighted" caption="Show Logs button" width="500px" align="center" src="/media/docs/explore-logs/show-logs.png" >}}
 
-From there, select **JSON** in the Logs format menu. This will show your logs in a structured, collapsible way, enabling you to sort, filter, and otherwise adjust your log data in the visualizations for your logs.
+You can use the **Line wrapping** control in the log controls to enable JSON formatting. When enabled with JSON formatting, your logs will be displayed in a structured, collapsible way, enabling you to sort, filter, and otherwise adjust your log data in the visualizations for your logs.
+
+The line wrapping control offers three options:
+- **Disabled**: Log lines are truncated at the edge of the panel.
+- **Enabled**: Log lines wrap to multiple lines.
+- **Enabled with JSON formatting**: JSON logs are pretty-printed for easier reading.
 
 {{< figure alt="Show Logs button on a JSON logging service" width="900px" align="center" src="/media/docs/explore-logs/json-viewer.png" caption="The JSON viewer" >}}
 
+For more information about log controls, refer to [View logs](../view-logs/).
+
 ## Filtering log lines with the JSON view
 
-You can include and exclude specific log data from your visualizations by selecting the **Include/Exclude** icons next to a given label.
+You can include and exclude specific log data from your visualizations by clicking on individual log lines to open the **Log Details** panel. Within Log Details, you can:
+
+- Click the **Include/Exclude** (plus/minus) icons next to fields to add them to your filter.
+- Click the eye icon to select specific fields to display in the logs list.
+- View field statistics by clicking the stats icon.
```

Comment:
Does it make sense to talk about the opening the log details panel in the JSON view @gtk-grafana ?